__tests__/app.test.js

```diff
@@ -29,6 +29,40 @@ describe("/api/topics", () => {
     });
   });
 });
+describe("/api", () => {
+  describe("GET request", () => {
+    test("responds with json representation of all the available endpoints of the api", () => {
+      return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({ body: { endpoints } }) => {
+          expect(endpoints).toEqual(endpoints);
+        });
+    });
+  });
+});
+
+xdescribe("/api/articles/:article_id", () => {
```

Comment:
I would like to see the test suite without an `x` so that we know the code you are pushing to main passes all the tests